etna/datasets/tsdataset.py

```diff
@@ -398,6 +398,11 @@ def inverse_transform(self):
             for transform in reversed(self.transforms):
                 tslogger.log(f"Inverse transform {repr(transform)} is applied to dataset")
                 self.df = transform.inverse_transform(self.df)
+                columns_after = set(self.columns.get_level_values("feature"))
+                # TODO: return regressors after inverse_transform
```

Comment:
Leave this comment in the begging of the method and remove the new logic, we will fix it later